images/kube-webhook-certgen/Dockerfile

```
@@ -0,0 +1,27 @@
+#Builder stage
+FROM ppc64le/golang:1.25.3 AS builder
+
+ARG TARGETARCH=ppc64le
+ARG GOLANG_VERSION=1.25.3
+
+RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/kubernetes/ingress-nginx.git /ingress-nginx && \
+    cd /ingress-nginx && \
+    git checkout controller-v1.13.3 
+#This specific release contains kube-webhook-certgen v1.6.3
+
+WORKDIR /ingress-nginx/images/kube-webhook-certgen/rootfs
+
+RUN go mod tidy && \
+    CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} go build -a -o kube-webhook-certgen main.go
```

If we adopt the buildx way to build an image.. 
```suggestion
    CGO_ENABLED=0 go build -a -o kube-webhook-certgen main.go
```